### DISCUSSION TITLE
Different IP submask possible?

### DISCUSSION THREAD

**Discussion by thestargateproject** [hassio-addons/addon-unifi#451] [Q&A]
Hi all,

Can i setup this addon to use an different IP address?
The Cloudkey is now setup in a other IP submask range 172.x.x.x, the HA is on 192.x.x.x 
This is done for security reason, Management VLAN can only reach to the Cloudkey/Router so other LAN/IOT devices can't access to Cloudkey/Router.

**Comment by aidasb123** [comment]
@thestargateproject Did you get it sorted? I have similar issue with IP address. My HA is on IoT network 7.xx and my Unifi devices on home network 3.xx. I can login without any issues but all my devices shows as offline on Unifi network server. 
If this can be changed to different IP address it should work perfectly.
